### DISCUSSION TITLE
<Page> was created with unknown prop

### DISCUSSION THREAD

**Discussion by austins** [sveltejs/kit#5971] [General]
I'm getting a lot of errors in the console in the browser with `1.0.0-next.415`:

```
<Page> was created with unknown prop 'errors'
<Page> was created with unknown prop 'data'
<Layout> was created with unknown prop 'data'
<Error> was created with unknown prop 'data'
<Error> was created with unknown prop 'errors'
```

Is anyone else getting these?

**Comment by weepy** [comment]
Yes I get these

**Comment by pilcrowonpaper** [comment]
So the way you initialize Svelte components is by calling something like this:

```
new App({
   target: document.body,
   props: {
      prop1: "foo"
   }
})
```

If the `App` components takes in `prop1` (by exporting it), it will have the value of `foo`. If `prop1` isn't exported, it will throw a warning in the console. SvelteKit passes on `errors` and `data` as props but if the page component doesn't accept them, you'll get those warnings. So nothing to worry about.

But, does it appear in production? If so, I would like to remove them as well.
